### ISSUE TITLE
Declaration of showOnIndex() has changed

### ISSUE THREAD

**Issue by 1destiny** [froala/nova-froala-field#31]
```
Error: Declaration of Froala\NovaFroalaField\Froala::showOnIndex() should be compatible with Laravel\Nova\Fields\FieldElement::showOnIndex($callback = true)
```
In Nova 2.1 (latest release).

Fix in `Froala.php`:

```
    /**
     * Specify that the element should be hidden from the index view.
     *
     * @param  \Closure|bool  $callback
     * @return $this
     */
    public function showOnIndex($callback = true)
    {
        $this->showOnIndex = $callback;

        return $this;
    }
```

**Comment by 1destiny** [comment]
Didn't notice the pull requests to fix this.

Rather than the mod I suggested, I agree that the override `showOnIndex` should be removed from `Froala.php`.

**Comment by slavarazum** [comment]
Fixed - [3.1.0 release](https://github.com/froala/nova-froala-field/releases/tag/3.1.0)
